libraries/stdlib/js-ir/builtins/Char.kt

```diff
@@ -12,7 +12,9 @@ package kotlin
 // TODO: KT-35100
 //@Suppress("NON_PUBLIC_PRIMARY_CONSTRUCTOR_OF_INLINE_CLASS")
 //public inline class Char internal constructor (val value: Int) : Comparable<Char> {
-public class Char internal constructor(private val value: Int) : Comparable<Char> {
+@OptIn(ExperimentalUnsignedTypes::class)
+public class Char(code: UShort) : Comparable<Char> {
```

Comment:
We will need primary constructor with property when we will make Char inline class again.

Could we keep primary constructor as is, and make UShort one secondary: 
```
public constructor(code: UShort) : this(chode.toInt())
```
?